js-miniapp-bridge/src/types/points.ts

```diff
@@ -0,0 +1,6 @@
+/** Points type. */
+export interface Points {
+  pointsStandard?: number;
+  pointsTerm?: number;
+  pointsCash?: number;
```

Comment:
Maybe these could be shorted to just:
```suggestion
  standard?: number;
  term?: number;
  cash?: number;
```
What do you think?